streaming/streaming.go

```diff
@@ -367,12 +368,18 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		inputFilePattern = inputFilePattern + "/"
 	}
 	fmt.Println("Reading files from datastream destination ", inputFilePattern)
+	var dataflowHostProjectId string
+	if dataflowCfg.HostProjectId == "" {
+		dataflowHostProjectId, _ = utils.GetProject()
```

Comment:
I think it will fail in case if GetProject returns empty string which it might if project is not set in env variable and gcloud config

Reply:
Is returning an empty string from `GetProject()` a valid return value? Shouldn't it return a non-empty project string, or an error otherwise?

Reply:
A bug has been created to handle the error with GetProject() separately : [b/273418828](https://b.corp.google.com/273418828)